webapp/components/create_comment.jsx

```diff
@@ -150,6 +151,8 @@ class CreateComment extends React.Component {
         Client.createPost(
             post,
             (data) => {
+                AsyncClient.getPosts(this.props.channelId);
```

Comment:
Ok, I'm fine with adding it back in for now but we should look into removing them since there isn't any reason we should need to do that. We don't do it in [create_post](https://github.com/mattermost/platform/blob/master/webapp/components/create_post.jsx#L183)

I made a ticket [here](https://mattermost.atlassian.net/browse/PLT-3221) for it